modules/atomic-widgets/module.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Elementor\Modules\AtomicWidgets;
+
+use Elementor\Core\Base\Module as BaseModule;
+use Elementor\Core\Experiments\Manager as Experiments_Manager;
+use Elementor\Modules\AtomicWidgets\Widgets\Atomic_Heading;
+use Elementor\Plugin;
+use Elementor\Widgets_Manager;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Module extends BaseModule {
+	const EXPERIMENT_NAME = 'atomic_widgets';
+	const EXPERIMENT_DEFAULT_STATE = Experiments_Manager::STATE_INACTIVE;
+
+	const PACKAGES = [
+		'editor-documents', // TODO: NEED to be removed once the editor will not be dependent on the documents package.
+		'editor-panels',
+	];
+
+	public function get_name() {
+		return 'atomic_widgets';
+	}
+
+	public function __construct() {
+		parent::__construct();
+
+		$this->register_experiment();
+
+		if ( Plugin::$instance->experiments->is_feature_active( self::EXPERIMENT_NAME ) ) {
+			add_filter( 'elementor/editor/v2/packages', fn( $packages ) => $this->add_packages( $packages ) );
+			add_filter( 'elementor/widgets/register', fn( Widgets_Manager $widgets_manager ) => $this->register_widgets( $widgets_manager ) );
+		}
```

Comment:
Just to make the construct clearer
```suggestion
$this->add_filters()
```

Reply:
I think it is better to be visible in the constructor.